src/main/java/com/tabnine/general/StaticConfig.java

```diff
@@ -16,227 +16,230 @@
-  public static final Icon ICON_AND_NAME_CONNECTION_LOST_STARTER =
-      IconLoader.findIcon("/icons/tabnine-starter-connection-lost-13px.png");
-  public static final Icon ICON_AND_NAME_CONNECTION_LOST_PRO =
-      IconLoader.findIcon("/icons/tabnine-pro-connection-lost-13px.png");
-  public static final Icon ICON_AND_NAME_CONNECTION_LOST_ENTERPRISE =
-      IconLoader.findIcon("/icons/tabnine-enterprise-connection-lost-13px.png");
-  public static final String LIMITATION_SYMBOL = "🔒";
-  public static final Color PROMOTION_TEXT_COLOR = decode("#e12fee");
-  public static final Color PROMOTION_LIGHT_TEXT_COLOR = decode("#FF99FF");
-  private static final int MAX_SLEEP_TIME_BETWEEN_FAILURES = 1_000 * 60 * 60; // 1 hour
-  public static final long BINARY_PROMOTION_POLLING_INTERVAL = 2 * 60 * 1_000L; // 2 minutes
-  public static final long BINARY_PROMOTION_POLLING_DELAY = 10_000L; // 10 seconds
-
-  public static final String OPEN_HUB_ACTION = "OpenHub";
-
-  // As far as I understand, This registers `BRAND_NAME` display id as a "sticky balloon" type,
-  // so that when we create a notification with this display id it knows to make it a sticky
-  // balloon.
-  public static NotificationGroup TABNINE_NOTIFICATION_GROUP =
-      new NotificationGroup(StaticConfig.BRAND_NAME, NotificationDisplayType.STICKY_BALLOON, false);
-
-  public static Optional<String> getLogFilePath() {
-    String logFilePathFromUserSettings = AppSettingsState.getInstance().getLogFilePath();
-    if (!logFilePathFromUserSettings.isEmpty()) {
-      return Optional.of(logFilePathFromUserSettings);
-    }
-
-    return Optional.ofNullable(System.getProperty(LOG_FILE_PATH_PROPERTY));
-  }
-
-  public static Optional<String> getLogLevel() {
-    String logLevelFromUserSettings = AppSettingsState.getInstance().getLogLevel();
-    if (!logLevelFromUserSettings.isEmpty()) {
-      return Optional.of(logLevelFromUserSettings);
-    }
-
-    return Optional.empty();
-  }
-
-  public static Optional<String> getTabnineEnterpriseHost() {
-    String path = AppSettingsState.getInstance().getCloud2Url();
-    if (!path.isEmpty()) {
-      return Optional.of(path);
-    }
-    return Optional.ofNullable(System.getProperty(TABNINE_ENTERPRISE_HOST));
-  }
-
-  public static String getServerUrl() {
-    return Optional.ofNullable(System.getProperty(REMOTE_BASE_URL_PROPERTY))
-        .orElse("https://update.tabnine.com");
-  }
-
-  public static Optional<String> getBundleServerUrl() {
-    if (Config.IS_SELF_HOSTED) {
-      Optional<String> tabnineEnterpriseHost = StaticConfig.getTabnineEnterpriseHost();
-      if (!tabnineEnterpriseHost.isPresent()) {
-        Logger.getInstance(StaticConfig.class).warn("On prem version but server url not set");
-        return Optional.empty();
-      }
-      return Optional.of(String.join("/", tabnineEnterpriseHost.get(), "update", "bundles"));
+    // Must be identical to what is written under <id>com.tabnine.TabNine</id> in plugin.xml !!!
```

Comment:
please revert all the format changes here, its confusing to understand the actual diff

Reply:
You can hide whitespaces diff... but will revert